tests/unittests/tests-kernel_defines/tests-kernel_defines.c

```diff
@@ -28,7 +28,7 @@ static void test_kernel_version(void)
     uint16_t mayor, minor, patch, extra;
 
     TEST_ASSERT(RIOT_VERSION_CODE);
-    TEST_ASSERT(RIOT_VERSION_CODE > KERNEL_VERSION(2020, 7, 1, 0));
+    TEST_ASSERT(RIOT_VERSION_CODE > KERNEL_VERSION(2017, 1, 3, 0));
     TEST_ASSERT(RIOT_VERSION_CODE < KERNEL_VERSION(2080, 10, 2, 0));
```

Comment:
Some poor release manager in 60 years will curse those developers of old ;-P

Reply:
They can update the test case :wink:

Reply:
That would also be better for the test case here. Just reading the test, 2017.01 seems somewhat random? Why not use 0.0?

Reply:
not that random if you consider the CI
```
main(): This is RIOT! (Version: 2017.04-devel)
```

Reply:
Yes... See https://github.com/RIOT-OS/RIOT/pull/16765#discussion_r712351267

Reply:
Still, if you do not know that, one might wonder why ;-).